addons/web/static/src/js/services/config.js

```diff
@@ -65,6 +66,12 @@ var config = {
          * Mapping between the numbers 0,1,2,3,4,5,6 and some descriptions
          */
         SIZES: { XS: 0, VSM: 1, SM: 2, MD: 3, LG: 4, XL: 5, XXL: 6 },
+        /**
+         * OdooMobile (iOS) App detection using userAgent.
+         *
+         * @return Boolean
+         */
+        isOdooMobileOS: isOdooMobileOS,
```

Comment:
For consistency with future versions, could you rename it to `isIosApp` (as defined in `feature_detection.js` since v15)